PluginGenerator/Resources/Rules/PluginRulesDefinition.java

```diff
@@ -22,21 +22,19 @@
   private static final Logger LOG = LoggerFactory.getLogger(PluginRulesDefinition.class);
 
   // Unique to the plugin: used when naming the repository and the properties
-  protected static final String KEY = "[PLUGIN_KEY]";
+  protected static final String REPOSITORY_ID = "[RULE_REPOSITORY_ID]";
```

Comment:
The [PLUGIN_KEY] symbol was being used for a number of logically different purposes. I've introduced this new symbol ([RULE_REPOSITORY_KEY]) to make it clearer what the value represents.